plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -206,3 +212,13 @@ func (l *lifecycle) Validate() error {
 	}
 	return nil
 }
+
+// accessReviewResources are resources which give a view into permissions in a namespace.  Users must be allowed to create these
+// resources because returning "not found" errors allows someone to search for the "people I'm going to fire in 2017" namespace.
+var accessReviewResources = map[schema.GroupResource]bool{
+	schema.GroupResource{Group: "authorization.k8s.io", Resource: "localsubjectaccessreviews"}: true,
```

Comment:
Would creating a `SelfSubjectAccessReview` for random namespaces leak information?